keps/sig-node/3063-dynamic-resource-allocation/README.md

```diff
@@ -1271,11 +1273,11 @@ type ResourceClaimStatus struct {
 	// +optional
 	DriverName string
 
-	// Allocation is set by the resource driver once a resource has been
-	// allocated successfully. If this is not specified, the resource is
-	// not yet allocated.
+	// Allocation is set by the resource driver once a (set of) resources has
+	// been allocated successfully. If this is not specified, the resources
+	// have not been allocated yet.
 	// +optional
-	Allocation *AllocationResult
+	Allocation []AllocationResult
```

Comment:
I suspect we will have to choose a list type here, and it probably has to be `+listType:atomic`: only the resource driver controller will make changes to this field, and it will always be all-or-nothing.

In other words, there's no need to patch individual entries. If there was, we would have to use `KubeletPluginName` as unique key.

Reply:
Agreed. It should be all-or-nothing. I can't imagine a case where something would need to change once it is set.

Is `+listType:atomic` something I add to the struct spec here (just like `+optional`)?

Reply:
Yes.

Reply:
Updated